app/uk/gov/hmrc/agentoverseasfrontend/views/application/personal_details.scala.html

```diff
@@ -51,14 +51,20 @@
     }
 
     @h1("personalDetails.title")
-    @p("personalDetails.p1", classes = Some("margin-bottom-10"))
 
-    <p class="govuk-hint">@msgs("personalDetails.p2")</p>
+    @p("personalDetails.p1", classes = Some("margin-bottom-10"))
 
     @formWithCsrf(action = routes.ApplicationController.submitPersonalDetails, 'class -> "form js-form") {
 
-        <fieldset class="govuk-fieldset">
         @govukRadios(Radios(
+            fieldset = Some(Fieldset(
+                legend = Some(Legend(
+                    content = Text(msgs("personalDetails.title")),
+                    classes = "govuk-visually-hidden"
+                ))
+            )),
+            name = "personalDetails",
```

Comment:
this name should not be added, it is automated by `withFormField` and is actually different, it's `personalDetailsChoice` which means this new name breaks the validation